proposals/project-smoot.md

```diff
@@ -0,0 +1,136 @@
GgHN49W4/k70eawss7/8BjxYlE178FIlqaBwN1/L7wvKnrpqqxW/RMnwULIdZN6TYKy8Bt32r9dPqanJrq/kvXc7oHjdPrYf/fb4Dh4OcPp6ZPgS4buOj6ZaAn3ZbOd6ITA3ecNE06ksZOfovf4c3eReuli/Hu4t73un2CDJf6jqZeRNZ/7v4ZtUeBe4g/Yh//xdk/Ket1SdF+7/veD25ngGO1Ln89G23mC7ulrJHBL/u2C7vJ60VUC58Xqun+p/NHo565xesq4DO26RvDlbn5doucJvWYaryy3/a930j45Lz7P6G2dvOz6zbz0qH9/JXUo+FId3AbwqGnv6bj4u5J0XERlObwto52AwKUUSV9WjSNwV75v3UdDG+1+pJ76/js6NLrxiJ6A8hBZCZzBPzldD++PCdeFAqWlyW989VbOf8DUyPLzJIEjknqiqJfUJvofXLLAGcLhE+zXGqsLnXD3W0/gRku/VLt2z16w//BXcgS/9yTo/XLD4u63XrSDm0db4Yuqfj/2fUwUVkOraSBwtkRROfF3IQzT6LpQj51dFsP+d3DbPyHTMpZ1G/s7Ze+bTuS3v2+W00mA0MfrXybWWN8NWyjcdbZf2N/1/UZmAnfI8dL2l2rvQ+7huq7LvfXPyfz44Okp83rCaXv2H+E/BPF1rYdmHz2t3SVwfo/mV196rmP+4bbbUrtOxnrDdY+W86MSH12/GggcPdf/yH5m7q01+0zOS28by8uU3em36Z/tmmkP/CtC1LEwvGzeqX1Zm1/XFTif8Acz7QUELqWIf1ndynZDbVv3R9Ayt5tYoy+nQeA4krj7I3td2POgP0//ciRh3+xM8I8ViHoCF55szQlq2L+h3r6M1brARSUiDYEjKu+ayzmGcP/ExmvmMdJLaO2Lo5sFjv6zvTIV3bcpy9Z64CZ+bErC3t6WKN0DF3wmZb+n1aQrz4bpnvIbV7csbHQ6/R+6uZRC8cq74WUxomw10pR2dz0qnqvru/rmdTtvTh/rgUsQuNPPmpMJ9VDxuv2EVIHTn89nn5oeerqXUvFtBfG08TBtcPmH5h/u8gf1e8WaBfXCR+7BO0DoY7K+o82imwRO/5gleG0Eqp7ATf4o/Ke58n60PnH7T/H8lP88Jl1ufQn/QbP3Geb1VtO8dDuwve6LnEGvP0APnN1utBMQuJSC/tsj9j42X5gr/qUxc0nG/Odx923vv78p88vHEm3n/+Lqlf9YDhqY7R/Tf4gQOIrhpfjlRburO4moXNmEFYXh53xyf9rvKg/5MpABF0ofXBZSpjwjDZZzLwXBJ1hNiwJnw7/eY/hiEW/Dl0vDhiR+rx5hX0J191sv2sHNo9W4veP8923dP5h0PyIF309CbC+FvZEUhPtDggi7/o8kLpq6ydy/a3prXIHjewztZS683P78zTL/tgn/Hzsuh7BnJnrPIhHZR5LAJR3jPkKqwFFsf+x//l99qe6+G//1asO4WNa/IiboHjoW+n3F901bvrf7aSo/wNpvdJPAnb5o9XJ/8WlwWZnWEbbAvbIVb7fMumj9sO9ttltqztOuLxt+WXKb99L70fbevU9a47elLJ+R84v14yp3KJxX/PvZGwkcS2zlTvuXUSFwKYZ7ctnd4hPp9eTxiB6NN/ymkkPgKLSQWDf5mjCiS+MqGT5V9/2T1/Z6tALS5RL9AxEV9qBRlLei9zTYJ/fbH5h19i/cOP1LXqUcvvZbfTIw626YfX+xp648m9Bg+bJO6HvorGNrV+Bees9Irb7R+1nzmXNvG/cWXnHyosuuGu+4+XPoZoFDdDYkC9xRj8MUuIMEUb5sfswyPGPuh2y1rTpKAYFDpBJZCFyzMMR/DcbrWr/fRUgkdP2nFe3g5oHozoDAyY08C9z2svnRF1/NOuj9gxIDAodIJSBwhxj+zf2EPdhsmtEObh6I7gwInNzIq8BNXPul2v3cvy3pc3uILYQdEDhEKnEYAofoXLSDmweiOwMCJzfyKnCI1gICh0glIHCyox3cPBDdGWkLXBJumoPGS7/ai9xHmsagqMOX4gM8P9/khwv2kDTuPbCE++OZTgcETnZA4BCpBAROdrSDmweiOyMLgYuPs5du2OPjpRWMnrHhYjn4UZqbzg4IHOIwAwKHSCUgcLKjHdw8EN0ZnRE4fwgJHhfT+sFN0swGPF4bD/0QsPtRLD39OtEM12Luh41PtxQdx1FFBsLm76n5pTb/otsNF85vvwL3ij/cVIh/D+9r5hfk9rb2WJU2SYN06yniEgICJzsgcIhUAgInO9rBzQPRnZGFwEXQ0tZc4Hh7PRWcNSC3Pe1ZZBgcqwfOFjiG18UGQw7WmVk+tPw0+QV3+f3oHMwsZvsVOIqla/5nZ8+N66fXg/ReNG2pXu4PEsvv26ipGcaIsKeISwoInOyAwCFSCQic7GgHNw9Ed0a3CZw7fVi8N68FgbPWufkljsHYRODcKewOInA31sOR/N30NO5l5PPwBzN3aZS/HRA42QGBQ6QSEDjZ0Q5uHojujCwELi5dvsD5A3PbM6o0E7hQuMrq7ttmGrmmAmflF+uBSxI4XmcPHP6sma+Ter+IpCn39itwPFuKfu1I4yvW5LZ775sx0LgHjtNMvpHcE1gvIHCyAwKHSCUgcLKjHZ56YRmRg+iMwDnfIb83i5Y3ErjYPXA7RmB41hGCBnjd1z1wdQRu292XSr7fzJ7CTs+rq8z7tZ+72zA8fZSBc+KxLM0sMwTP95uYj3V5OSuBc78jiO6NbiFR4AAAAHQXjQQO0X4s0ZSasekD2492BQ6A/QKBAwCAHACBSzdGn11SwzPmMrM7l/FBAgIHOgUEDgAAcgAELt3Y2DX3BN5+7Z3YuoMEBA50CggcAADkAAhcPgICBzoFBA4AAHIABC4fAYEDnQICBwAAOQACl4+AwIFOAYEDAIAcAIHLR0DgQKeAwAEAQA6AwOUjIHCgU0DgAAAAAAByxv8PaE8o7UewvXsAAAAASUVORK5CYII=>
```

Comment:
I recommend moving these to distinct files in the repository.  There is typically an `images` folder you can use (and or create)